Month1/Week2/Section2/Month1-Week2-Section2/Spotify/Song.cpp

```diff
@@ -0,0 +1,70 @@
+#include "Song.h"
+FSong::FSong()
+{
+    MinutesDuration = 0;
+    SecondsDuration = 0;
+    ArtistName = "";
+    SongName = "";
+}
+FSong::FSong(int EntryMinutesDuration, int EntrySecondsDuration, std::string EntryArtistName, std::string EntrySongName)
+{
+    MinutesDuration = EntryMinutesDuration;
+    SecondsDuration = EntrySecondsDuration;
+    ArtistName = EntryArtistName;
+    SongName = EntrySongName;
+
+}
+
+
+
+std::string FSong::GetFormatedDuration()const
+{
+    std::string FormatedDuration = "";
+    FormatedDuration = "Minutes: " + std::to_string(MinutesDuration) + "\t" "Seconds" + std::to_string(SecondsDuration);
+    return FormatedDuration;
+}
+bool FSong::bIsvalid()
+{
+    if (MinutesDuration <= 0 && SecondsDuration < 0)
```

Comment:
This code works perfectly fine, but it is a bit complex for how trivial it is. Instead of writing
``` 
if (bCondition)
{
    return true;
}
else
{
    return false;
}
```
you could just write
```
return bCondition;
```
So, in this case, you could just write:
```
return MinutesDuration > 0 && SecondsDuration >= 0;
```

That way your function is clearer and easier to read, and you saved 7 lines for such a simple function.

Reply:
Wow, I didn't know that method, I really thank you